test/test_autopep8.py

```diff
@@ -5378,6 +5378,17 @@ def test_standard_in(self):
             fixed,
             process.communicate(line.encode('utf-8'))[0].decode('utf-8'))
 
+    def exit_code_should_be_set_when_standard_in(self):
```

Comment:
I would like the test method name prefix to be `test_`
```suggestion
    def test_exit_code_should_be_set_when_standard_in(self):
```

Reply:
Sure! I'm amending the commit.